index.html

```diff
@@ -41,8 +41,29 @@ <h2> Skills</h2>
 <section>
 <h1>Connect </h1>
 <a href= https://www.linkedin.com/in/kipardillo/> linkedin </a>
+</section>
+
+<section>
+    <h2> Leave a Message</h2>
+    <form name = "leave_message">
+        <label for  = leave_message> </label>
+        <input type = "text" name="name" required = true> <br><br>
+        <input type = "email" name="email" required = true > <br><br>
+        <textarea name = "message"  required = true>
+        </textarea> <br><br>
+        <button type = "submit"> Submit </button>
+    </form>
```

Comment:
Spacing for attributes is not needed. Also just a fun cool thing about HTML syntax that you might see around: if your attribute is `="true"` , you can just put the attribute name and HTML will interpret it as such. So this could be written as:

```
    <form name="leave_message">
        <label for=leave_message> </label>
        <input type="text" name="name" required> <br><br>
        <input type="email" name="email" required> <br><br>
        <textarea name="message" required>
        </textarea> <br><br>
        <button type="submit"> Submit</button>
    </form>
```